cmake/modules/FindTAEF.cmake

```diff
@@ -27,15 +27,7 @@ macro(find_taef_libraries targetplatform)
   set(TAEF_COMMON_LIBRARY ${TAEF_LIB_Te.Common.lib})
 endmacro(find_taef_libraries)
 
-if ("${DXC_BUILD_ARCH}" STREQUAL "x64" )
-  find_taef_libraries(x64)
-elseif (CMAKE_GENERATOR MATCHES "Visual Studio.*ARM" OR "${DXC_BUILD_ARCH}" STREQUAL "ARM")
-  find_taef_libraries(arm)
-elseif (CMAKE_GENERATOR MATCHES "Visual Studio.*ARM64" OR "${DXC_BUILD_ARCH}" MATCHES "ARM64.*")
-  find_taef_libraries(arm64)
-elseif ("${DXC_BUILD_ARCH}" STREQUAL "Win32" )
-  find_taef_libraries(x86)
-endif ("${DXC_BUILD_ARCH}" STREQUAL "x64" )
+find_taef_libraries(${CMAKE_C_COMPILER_ARCHITECTURE_ID})
```

Comment:
Could you please check if hctbuild -arm64ec works? I suspect it might not in this case. There is no arm64ec or arm64x directory in WinSDK, we should use the arm64 one.

Reply:
I'm struggling to get arm64ec to build locally without this change. I'm sure I don't have my system setup correctly :(